Doc/library/zipfile.rst

```diff
@@ -325,7 +326,7 @@ ZipFile Objects
    must be its full name or a :class:`ZipInfo` object.  Its file information is
    extracted as accurately as possible.  *path* specifies a different directory
    to extract to.  *member* can be a filename or a :class:`ZipInfo` object.
-   *pwd* is the password used for encrypted files.
+   *pwd* is the password used for encrypted files as a :class:`bytes`.
```

Comment:
nit: consistency with the other phrasings
```suggestion
   *pwd* is the password used for encrypted files as a :class:`bytes` object.
```

Reply:
Thanks for catching that! Fixed.